resource-server/okta-aspnet-webapi-example/Startup.cs

```diff
@@ -13,8 +13,7 @@ public void Configuration(IAppBuilder app)
         {
             app.UseOktaWebApi(new OktaWebApiOptions()
             {
-                OktaDomain = ConfigurationManager.AppSettings["okta:OktaDomain"],
-                ClientId = ConfigurationManager.AppSettings["okta:ClientId"],
```

Comment:
I was getting a warning that the `okta:ClientId` setting is deprecated, so I removed it.  Not sure if there's something else I should do with it instead?

Reply:
You did right. This is one of the changes we made recently in our ASP.NET SDK and was not reflected in our samples 👍